ui/apps/platform/src/constants/accessControl.ts

```diff
@@ -85,3 +87,49 @@ export const defaultSelectedRole = {
     name: '',
     resourceToAccess: defaultNewRolePermissions,
 };
+
+// TODO: ROX-12750 update with new list of replaced/deprecated resources
+export const resourceSubstitutions: Record<string, string[]> = {
+    Access: ['AuthProvider', 'Group', 'Licenses', 'User'],
+    DeploymentExtension: ['Indicator', 'NetworkBaseline', 'ProcessWhitelist', 'Risk'],
+    Integration: [
+        'APIToken',
+        'BackupPlugins',
+        'ImageIntegration',
+        'Notifier',
+        'SignatureIntegration',
+    ],
+    Image: ['ImageComponent'],
+};
+
+// TODO: ROX-12750 update with new list of replaced/deprecated resources
+export const resourceRemovalReleaseVersions = new Map<ResourceName, string>([
+    ['AllComments', '3.75'],
+    ['ComplianceRuns', '3.75'],
+    ['Config', '3.75'],
+    ['DebugLogs', '3.75'],
+    ['NetworkGraphConfig', '3.75'],
+    ['ProbeUpload', '3.75'],
+    ['ScannerBundle', '3.75'],
+    ['ScannerDefinitions', '3.75'],
+    ['SensorUpgradeConfig', '3.75'],
+    ['ServiceIdentity', '3.75'],
+    ['ClusterCVE', '3.74'],
+]);
+
+// TODO(ROX-11453): Remove this mapping once the old resources are fully deprecated.
+export const replacedResourceMapping = new Map<ResourceName, string>([
+    ['AllComments', 'Administration'],
+    ['ComplianceRuns', 'Compliance'],
+    ['Config', 'Administration'],
+    ['DebugLogs', 'Administration'],
+    ['NetworkGraphConfig', 'Administration'],
+    ['ProbeUpload', 'Administration'],
+    ['ScannerBundle', 'Administration'],
+    ['ScannerDefinitions', 'Administration'],
+    ['SensorUpgradeConfig', 'Administration'],
+    ['ServiceIdentity', 'Administration'],
+    ['ClusterCVE', 'Cluster'],
+]);
+
+export const deprecatedResourceRowStyle = { backgroundColor: 'rgb(255,250,205)' };
```

Comment:
We will follow up with a PatternFly variable if explicit color does not work for dark mode